docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several reasons why we built .NET Core.
+
+**Cross-platform support** is a requirement of many developers today. It's important to use the 
+platform that has the characteristics required by the application, which, in turn, can run on the 
+platform required by users or perhaps is prescribed by corporate policy.
+
+**Open Source** because it has proven to be the best possible way to enable a larger set of 
+platforms, supported by community contribution.
+
+**Better packaging story** of the entire runtime is also something that influenced the 
+design of the runtime. We wanted to make sure that the entire runtime can be distributed 
+as a set of pieces that developers can pick and choose from, rather than a single, 
+monolithic platform. .NET Core is the first implementation of .NET Platform that will be 
+distributed via `NuGet <http://www.nuget.org/>`_ package manager. 
+
+**Better isolation and versioning** was one of the influencers to create .NET Core. Today's 
+trends in development practices lead to a very different way of developing sofware 
+than what was common even a couple of years ago. Cloud services are posed to radically 
+change the way developers build, test and run their solutions. Application containers like Docker are introducing new 
+possibilities and opportunities when it comes to server (and services) consolidation. With 
+this in mind, the .NET Core is targeting scenarios where the application, essentially, "takes" 
+the runtime with it. This allows for per-app versioning scenario; the developer needs to worry 
+only about the runtime (a set of packages, as seen in the previous paragraph) that her app 
+is dependent on. 
+
+With all of the above, .NET Core represents a very generic, cross-platform and modern 
```

Comment:
You probably don't need to repeat this. It's very similar to the intro paragraph.